src/sentry/models/integrations/repository_project_path_config.py

```diff
@@ -17,7 +17,7 @@ class RepositoryProjectPathConfig(DefaultFieldsModel):
     source_root = models.TextField()
     default_branch = models.TextField(null=True)
     # Indicates if Sentry created this mapping
-    automatically_generated = models.BooleanField(default=False)
+    automatically_generated = models.BooleanField(null=True, default=False)
```

Comment:
The column doesn't have to be null here. You can follow this guide to make the column not null with a default https://develop.sentry.dev/database-migrations/#adding-columns-with-a-default